docs/docs/30-installation/02-options.md

```diff
@@ -31,6 +31,29 @@ Let's Encrypt integration is only useful if you are running a non-local Kubernet
 ## Endpoint domain names
 Acorn provides several installation options for controlling the domain name used to generate endpoints. These are outlined in detail on our [networking page](/running/networking#dns).
 
+## Custom CA bundle
+Acorn allows user to provide custom ca bundle so that users can add own private CA and have acorn trust that CA. The most common use case is for acorn to trust an internal image registry that is signed by private CA.
```

Comment:
nit:
```suggestion
Acorn allows a user to provide a custom certificate authority (CA) bundle so that users can add their own private CA that acorn will trust. The most common use case is for acorn to trust an internal image registry that is signed by a private CA.
```